ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -178,9 +184,9 @@ private[kvutils] class PackageCommitter(
       submissionId: String,
       participantId: String,
       addErrorDetails: DamlPackageUploadRejectionEntry.Builder => DamlPackageUploadRejectionEntry.Builder,
-      incrementMetric: Boolean = true,
+      incrementRejectionCount: Boolean = true,
```

Comment:
I still feel like a less technical flag name would be more readable, like `dryRun` or `speculative` (so as not to tie this to `preexecution`) but there are less calls to this method now, so I'm OK with this.